sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -0,0 +1,444 @@
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN preferences_jump_back_in > 0
+          THEN client_id
+      END
+    ) AS preferences_jump_back_in_users,
+    COALESCE(SUM(preferences_jump_back_in), 0) AS preferences_jump_back_in,
+    -- Preferences Recently Visited
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN preferences_recently_visited > 0
+          THEN client_id
+      END
+    ) AS preferences_recently_visited_users,
+    COALESCE(SUM(preferences_recently_visited), 0) AS preferences_recently_visited,
+    -- Preferences Recently Saved
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN preferences_recently_saved > 0
+          THEN client_id
+      END
+    ) AS preferences_recently_saved_users,
+    COALESCE(SUM(preferences_recently_saved), 0) AS preferences_recently_saved,
+    -- Preferences Pocket
+    COUNT(DISTINCT CASE WHEN preferences_pocket > 0 THEN client_id END) AS preferences_pocket_users,
+    COALESCE(SUM(preferences_pocket), 0) AS preferences_pocket,
+    -- App Menu Customize Homepage
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN app_menu_customize_homepage > 0
+          THEN client_id
+      END
+    ) AS app_menu_customize_homepage_users,
+    COALESCE(SUM(app_menu_customize_homepage), 0) AS app_menu_customize_homepage,
+    -- Firefox Home Page Customize Homepage Button
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN firefox_home_page_customize_homepage_button > 0
+          THEN client_id
+      END
+    ) AS firefox_home_page_customize_homepage_button_users,
+    COALESCE(
+      SUM(firefox_home_page_customize_homepage_button),
+      0
+    ) AS firefox_home_page_customize_homepage_button
+  FROM
+    product_features
+  WHERE
+    submission_date >= start_date
+  GROUP BY
+    1
+)
+SELECT
+  submission_date,
+  dau
+/*Credit Card*/
+  ,
+  credit_card_autofill_enabled_users,
+  credit_card_autofill_enabled,
+  credit_card_sync_enabled_users,
+  credit_card_sync_enabled
+/*Bookmark*/
+  ,
+  bookmarks_add_users,
+  bookmarks_add,
+  bookmarks_delete_users,
+  bookmarks_delete,
+  bookmarks_edit_users,
+  bookmarks_edit,
+  has_mobile_bookmarks_users,
+  has_mobile_bookmarks,
+  mobile_bookmarks_count_users,
+  mobile_bookmarks_count,
+  bookmarks_open_users,
+  bookmarks_open,
+  bookmarks_view_list_users,
+  bookmarks_view_list
+/*FxA*/
+  ,
+  sync_create_account_pressed_users,
+  sync_create_account_pressed,
+  sync_open_tab_users,
+  sync_open_tab,
+  sync_sign_in_sync_pressed_users,
+  sync_sign_in_sync_pressed
+/*Privacy*/
+  ,
+  tabs_private_tabs_quantity_users,
+  tabs_private_tabs_quantity,
+  preferences_close_private_tabs_users,
+  preferences_close_private_tabs,
+  tracking_protection_enabled_users,
+  tracking_protection_enabled,
+  tracking_protection_strict_users,
+  tracking_protection_strict
+/*Tab Count*/
+  ,
+  tabs_normal_tabs_quantity_users,
+  tabs_normal_tabs_quantity,
+  tabs_inactive_tabs_count_users,
+  tabs_inactive_tabs_count
+/*Default Browser*/
+  ,
```

Comment:
Is there a specific reason why there's a comma on it's own line here? I expect this will fail formatting check.